src/docfx/build/resolve/Resolve.cs

```diff
@@ -21,30 +21,43 @@ public static (DocfxException error, string content, Document file) TryResolveCo
             return file != null ? (error, file.ReadText(), file) : default;
         }
 
-        public static (DocfxException error, string href, string fragment, Document file) TryResolveHref(this Document relativeTo, string href, Document resultRelativeTo)
+        public static (DocfxException error, string href, Document file, bool hasBookmark) TryResolveHref(this Document relativeTo, string href, Document resultRelativeTo)
```

Comment:
>bool hasBookmark [](start = 73, length = 16)

fragment doesn't means bookmark like #row=4?  it's better to returen the fragment, otherwise you need return more flags if we need